### PR TITLE
Add a file for Maple Lake, MN

### PR DESCRIPTION
**Pull request by ghost**
The purpose of this PR is to add a hometown file for Maple Lake, MN.

Some ? that I want you all to help me answer:

1. If you've been to Maple Lake, what did you do?  What did you think of it?
1. Have you even heard of Maple Lake?
1. What kind of food do you like?

### CODE REVIEW

**Comment by AdamPaolo** [comment]
1. Nope, I've never been.
2. Are Lakes better than Groves?  Food for thought.
3. I like lunch.

**Comment by ghost** [comment]
@AdamPaolo  2 - Regrettably not, in this context, 3 - This is something we can agree upon.

**Comment by ghost** [comment]
This PR resolves #5

File: upcrob-securian-maplelake.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+* The school
+* The VFW
+
+### Places tourists like
+
+* Tourists?
+
+### Miscellaneous
+
+There's some debate as to whether lakes are better than groves...
```
Comment (AdamPaolo) [inline upcrob-securian-maplelake.md]: There isn't a state with 10,000 Groves...

**Review by AdamPaolo — CHANGES_REQUESTED** [review]
NOT GOOD ENOUGH

File: upcrob-securian-maplelake.md (hunk 1 above)
Comment (ghost) [reply upcrob-securian-maplelake.md]: I expect this attitude from Iowa.

**Review by AdamPaolo — APPROVED** [review]

**Comment by AdamPaolo** [comment]
Much better.

**Pull request merged**